### PR TITLE
README.md!

### PR DESCRIPTION
**Pull request by allefant**

### CODE REVIEW

File: README.txt [hunk 1]
```diff
@@ -1,16 +1,23 @@
...
+playing sounds, etc. and generally abstracting away the underlying
+platform. However, Allegro is not a game engine: you are free to design
+and structure your program as you like.
+
+Allegro 5 has the following additional features:
+
+- Supported on Windows, Linux, Mac OSX, iPhone and Android
+- User-friendly, intuitive C API usable from C++ and many other languages
+- Hardware accelerated bitmap and graphical primitive drawing support (via OpenGL or Direct3D)
+- Audio recording† support
```
Comment (SiegeLord) [inline README.txt]: One more :).

**Comment by allefant** [comment]
Found it!

File: README.txt [hunk 2]
```diff
@@ -1,16 +1,23 @@
...
-
-Overview
-========
-
-Welcome to Allegro 5, a cross-platform game programming library.
-Currently supported platforms are Linux/Unix, Windows, MacOS X and iPhone.
-Note that Allegro 5 is wholly *incompatible* with Allegro 4 and earlier
-versions, but both may be installed at the same time without conflicts.
+Welcome to Allegro!
+===================
```
Comment (MarkOates) [inline README.txt]: Awesome! :smile:

File: README.txt [hunk 3]
```diff
@@ -1,16 +1,23 @@
...
+and structure your program as you like.
+
+Allegro 5 has the following additional features:
+
+- Supported on Windows, Linux, Mac OSX, iPhone and Android
+- User-friendly, intuitive C API usable from C++ and many other languages
+- Hardware accelerated bitmap and graphical primitive drawing support (via OpenGL or Direct3D)
+- Audio recording† support
+- Font loading and drawing
+- Video playback†
```
Comment (MarkOates) [inline README.txt]: ^ here to

File: README.txt [hunk 4]
```diff
@@ -1,16 +1,23 @@
...
+- Abstractions over shaders† and low-level polygon drawing
```
Comment (MarkOates) [inline README.txt]: ^ OMG †hey're eve†ywhere!

File: README.txt [hunk 5]
```diff
@@ -15,8 +15,8 @@ Allegro 5 has the following additional features:
 - Hardware accelerated bitmap and graphical primitive drawing support (via OpenGL or Direct3D)
 - Audio recording† support
```
Comment (MarkOates) [inline README.txt]: ^

**Comment by SiegeLord** [comment]
Merged as bb6d0aa.

**Comment by MarkOates** [comment]
Niiiiiiice :sunglasses: 

image

**Pull request closed without merging**